src/dpl/README.md

```diff
@@ -31,15 +39,23 @@ names, or an instance object returned by the SDC `get_cells` command. To
 specify padding for all instances of a common master, use the `-filter`
 "ref_name == <name>" option to `get_cells`.
 
-The `-max_displacement` argument to `detailed_placement` specifies how
-far an instance can be moved when finding a site where it can be placed. The default values are
-`{500 100}` sites. The x/y displacement arguments are in microns.
+```tcl
+set_placement_padding   
+    -global|-masters masters|-instances insts
+    [-right site_count]
+    [-left site_count]
+    [instances]
+```
 
-The `check_placement` command checks the placement legality. It returns
-`0` if the placement is legal.
+#### Options
 
-The `optimize_mirroring` command mirrors instances about the Y axis in
-a weak attempt to reduce total wirelength (HPWL).
+| Switch Name | Description | 
+| ----- | ----- |
+| `-global` or `-masters` or `-instances` | Either flag must be set. For `-global`, you will set padding globally using `left` and `right` values. For `-masters`, you will set padding only for these masters using `left` and `right` values. For `-instances`, you will set padding only for these insts using `left` and `right` values. |
```

Comment:
Explain each in their own row. Add a warning callout before the table to explain that only one of the three will be honored, and mention that the order of preference is `global > masters > instances`. If you need clarification, please ask mina1460.